docs/samples/nft.md

```diff
@@ -19,7 +19,7 @@ The NFT canister is not very complicated since the [DIP-721](https://github.com/
 but we can still use it to explain three important concepts concerning dapp development for the Internet Computer:
 
 ### Stable Memory for Canister Upgrades
-The Internet Computer employs [Orthogonal Persistence](https://smartcontracts.org/docs/current/developer-docs/build/languages/motoko#orthogonal-persistence), so developers generally do not need to think a lot about storing their data.
+The Internet Computer employs [Orthogonal Persistence](https://smartcontracts.org/docs/current/developer-docs/build/cdks/motoko-dfinity#orthogonal-persistence), so developers generally do not need to think a lot about storing their data.
```

Comment:
URL is broken and old, please make it point to an .md file

Reply:
fixed